packages/ui/src/components/flag/flag.native.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { View } from 'react-native';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Box } from '../box/box.native';
+import { Flag } from './flag.native';
+
+const meta: Meta<typeof Flag> = {
+  title: 'Layout/Flag',
+  component: Flag,
+  tags: ['autodocs'],
+  argTypes: {
```

Comment:
Ideally these controls would work in the simulator but we will need to investigate fixing that as [we disabled add ons](https://github.com/leather-io/mono/pull/381) for now so storybook would load